ado_asana_sync/sync/sync.py

```diff
@@ -295,19 +480,38 @@ def create_asana_task(a: app, asana_project: "str", task: "work_item"):
     body = asana.TasksBody(
         {
             "name": task.asana_title,
-            "due_on": task.due_date,
             "html_notes": f"<body>{task.asana_notes_link}</body>",
             "projects": [asana_project],
             "assignee": task.assigned_to,
         }
     )
     try:
-        tasks_api_instance.create_task(body)
+        result = tasks_api_instance.create_task(body)
```

Comment:
**Suggestion:** Consider adding a check to ensure that the `result` object returned by the `create_task` and `update_task` methods is not `None` before accessing its `data` attribute. This will prevent a `NoneType` error if the methods return `None`.
```suggestion
        result = tasks_api_instance.create_task(body)
        if result is not None:
            task.asana_gid = result.data.gid
        ...
        result = tasks_api_instance.update_task(body, task.asana_gid)
        if result is not None:
            task.asana_updated = iso8601_utc(result.data.modified_at)
```